azure-iot-hub-devicesdk/azure/iot/hub/devicesdk/inbox.py

```diff
@@ -0,0 +1,27 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+"""This module contains an Inbox class for use with a synchronous client."""
+
+from six.moves import queue
+
+
+class Inbox(queue.Queue):
+    """Queue-like object, used to recieve Messages.
```

Comment:
>recieve [](start = 34, length = 7)

nit: typo. most devtools have a spell-checker extension if you're worried about those.